vend_ax.c

```diff
@@ -0,0 +1,318 @@
+/***************************************************
```

Comment:
I'd rather avoid this patch completely. Not familiar enough with the licensing implications.

Reply:
This comes from the cypress program cyusb suite for linux, that is under LGPL.
In short: A combination of GPL code and LGPL code must be licensed under GPL.
LGPL code has the same license as GPL with the addition that you can build a library out of it and share it _WITHOUT_ the need to provide the source code. This is lost as soon GPL code is added.